nav2_world_model/include/nav2_world_model/world_model.hpp

```diff
@@ -55,9 +42,10 @@ class WorldModel : public rclcpp::Node
 
   // Server for providing a costmap
```

Comment:
By the way, in the costmap_callback prototype, the formal parameters don't need to be commented out. Only comment out in the implementation file if they aren't used.